backend/src/main/java/com/github/wekaito/backend/Card.java

```diff
@@ -0,0 +1,23 @@
+package com.github.wekaito.backend;
+
+import java.util.Optional;
+
+public record Card(
+        String name,
+        String type,
+        String color,
+        String image_url,
+        String cardnumber,
+        Optional<String> stage,
+        Optional<String> attribute,
+        Optional<String> digi_type,
+        Optional<Integer> dp,
+        Optional<Integer> play_cost,
+        Optional<Integer> evolution_cost,
+        Optional<Integer> level,
+        Optional<String> maineffect,
+        Optional<String> sourceeffect
```

Comment:
Optionals werden meist nur für Rückgabewerte in Methode benutzt und nicht als Eigenschaften von Klassen. 
Dann hast du auch kein Problem wenn du das Objekt Serialisieren und in der Datenbank speichern musst.